xarray/core/dataset.py

```diff
@@ -2788,6 +2959,26 @@ def tail(
             The keyword arguments form of ``indexers``.
             One of indexers or indexers_kwargs must be provided.
 
+        Examples
```

Comment:
Could use same example as for `.head` but first sort by descending instead of ascending? Or could come up with a completely different example.